tests/python/relay/test_pass_inline.py

```diff
@@ -0,0 +1,837 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+# pylint: disable=invalid-name, missing-docstring, too-many-statements
+import tvm
+from tvm import relay
+
+
+def get_recursive_count_loop():
+    mod = tvm.IRModule({})
+    sum_up = relay.GlobalVar('sum_up')
```

Comment:
Can we use while_loop here? 

https://github.com/apache/incubator-tvm/blob/1dbdcfb5e0542dbe43a468ad31a71a674a13e1fa/python/tvm/relay/loops.py#L24

Reply:
Yes we can, but I intentionally used recursive here because while_loop is just a common node in the function. Recursive function should be handled differently.

Reply:
I see, good point.